homework/test/test-list.js

```diff
@@ -1,8 +1,10 @@
 describe('Test > ', function() {
   beforeEach(function() {
+  
   });
 
   it('will test some pure functions', function() {
-    // Write any pure function assertion here.
+    assert.equal('2', list.add(1, 1));
```

Comment:
I know this looks matching the item in my criteria, but your module and exporting way really needs to be fixed. And, please note you are **comparing a string to a number**, which is dangerous and buggy.